os/arch/arm/src/amebasmart/amebasmart_pmhelpers.c

```diff
@@ -112,21 +112,32 @@ int SOCPS_AONWakeReason(void)
 	return reason;
 }
 
-void pg_timer_int_handler(void *Data)
+void up_set_pm_timer(unsigned int interval_us) 
 {
-	pmvdbg("PM Timer interrupt handler!!\n");
-}
-
-void up_set_pm_timer(unsigned int interval_us) {
 	// Check whether timer interrupt need to be set
 	if (interval_us > 0) {
 		gtimer_init(&g_timer1, TIMER1);
 		/* Pass in timer obj to avoid compile warning, the last argument will not be used in the callback handler */
-		gtimer_start_one_shout(&g_timer1, interval_us, (void *)pg_timer_int_handler, (uint32_t)&g_timer1);
+		gtimer_start_one_shout(&g_timer1, interval_us, NULL, (uint32_t)&g_timer1);
 	}
 	return;
 }
 
+void bsp_pm_domain_control(char *domain, bool is_suspend)
+{
+	/* Retrive the domain id from the domain string */
+	int domain_id = pm_domain_register(domain);
```

Comment:
i think its better if we create a  pm_get_domainId(domain_name) for this? , @gSahitya-samsung